lib/Wikia/src/Service/Helios/HeliosClientImpl.php

```diff
@@ -149,7 +149,7 @@ public function login( $username, $password )
 			[ 'method'	=> 'POST' ]
 		);
 
-		return $response;
+		return [$this->status, $response];
```

Comment:
The way it's coded the `$response` is the json decoded response object, and there's no way to get at the integer status. I agree that we should have some sort of `HttpResponse` object that contains everything we need, but I didn't want to make such a major change to the helios implementation usage in MW to fix piggybacking.